server/controllers/student/getQuiz.ts

```diff
@@ -0,0 +1,16 @@
+import { NextFunction, Request, Response } from 'express';
+import { getQuizQuery } from '../../database/queries';
+import { CustomError } from '../../errors';
+
+const getQuiz = async ({ params }:Request, res:Response, next:NextFunction) => {
+  try {
+    const { rows: { 0: quiz }, rowCount } = await getQuizQuery(params.quizId);
+
+    if (!rowCount) throw new CustomError('No quiz found with that join code', 404);
```

Comment:
I don't sure if this `404` status code is the appropriate status code for this case,
because `404` is used when you make a request for a specific endpoint and your server doesn't server that endpoint, in this case, the server responds with `404`
So I think it should be 200